src/Sylius/Behat/Element/Product/IndexPage/VerticalMenuElementInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Element\Product\IndexPage;
+
+interface VerticalMenuElementInterface
+{
+    public function getMenuItems(): array;
+
+    public function hasGoLevelUpLink(): bool;
```

Comment:
This function name is too detailed. Maybe something like `canNavigateToParentTaxon()`?